tracks/csharp/exercises/nucleotide-count/mentoring.md

```diff
@@ -0,0 +1,57 @@
+### Reasonable solutions
+
+### Using a foreach loop
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class NucleotideCount
+{
+    public static IDictionary<char, int> Count(string sequence)
+    {
+        var result = new Dictionary<char, int> { {'A', 0}, {'C', 0}, {'G', 0}, {'T', 0} };
+        
+        foreach (var c in sequence)
+        {
+            if (result.ContainsKey(c)) result[c]++;
+            else throw new ArgumentException("Invalid Nucleotide Symbol");
```

Comment:
```suggestion
            else
                throw new ArgumentException("Invalid Nucleotide Symbol");
```

Having the statement of the if-clause on the same line is not something that one sees used often in C#.  Maybe put it on the next line (like [bob](https://github.com/exercism/website-copy/blob/master/tracks/csharp/exercises/bob/mentoring.md))?

Reply:
Resolved in 61a32fd4aefa4877c053be4d36cc1b7bffa696b8